Sources/Sentry/include/SentryFormatter.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryFormatter : NSObject
```

Comment:
`m`: Maybe name it `SentryByteCountFormatter` as `SentryFormatter` is not specific. Can we also please add a comment on why we added this?